src/Operation/CreateCollection.php

```diff
@@ -170,6 +178,10 @@ public function __construct($databaseName, $collectionName, array $options = [])
             throw InvalidArgumentException::invalidType('"maxTimeMS" option', $options['maxTimeMS'], 'integer');
         }
 
+        if (isset($options['pipeline']) && ! is_array($options['pipeline'])) {
+            throw InvalidArgumentException::invalidType('"pipeline" option', $options['pipeline'], 'array');
+        }
```

Comment:
Please add new constructor option tests to [CreateCollectionTest.php](https://github.com/mongodb/mongo-php-library/blob/1.12.0/tests/Operation/CreateCollectionTest.php) for `pipeline` and `viewOn`, which expect exceptions for non-array and non-string values, respectively.

Additionally, please create a new `CreateCollectionTest::testConstructorPipelineOptionMustBeAList` test based on [AggregateTest::testConstructorPipelineArgumentMustBeAList](https://github.com/mongodb/mongo-php-library/blob/1.12.0/tests/Operation/AggregateTest.php#L10).